plugins/proxies/lib_dmslib.eclmod

```diff
@@ -0,0 +1,22 @@
+/*##############################################################################
```

Comment:
@richardkchapman should we just delete this plugin.  I don't think it is used by any queries.  Possibly remove in 7.8.0 instead?